cypress/fixtures/filings/addressChange/completed.ts

```diff
@@ -13,14 +13,14 @@ export const addressChange = {
   displayLedger: true,
   displayName: 'Address Change',
   documentsLink:
-    'https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/CP1002605/filings/149681/documents',
+    'https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/BC1002605/filings/149681/documents',
   effectiveDate: 'Tue, 02 Jul 2024 16:30:56 GMT',
   filingId: 149681,
-  filingLink: 'https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/CP1002605/filings/149681',
+  filingLink: 'https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/BC1002605/filings/149681',
   isFutureEffective: false,
   name: 'changeOfAddress',
   paymentStatusCode: 'COMPLETED',
-  status: 'COMPLETED',
+  status: 'PAID',
```

Comment:
This file is only used for a "BEN" test, Also, "addressPending" was recently updated to check for only PAID change of address filings.

Reply:
cc: @ArwenQin